group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -931,17 +920,14 @@ public void testGroupEpochBumpWhenNewStaticMemberJoins() {
 
         ConsumerGroupMember expectedMember3 = new ConsumerGroupMember.Builder(memberId3)
             .setMemberEpoch(11)
+            .setState(MemberState.UNRELEASED_PARTITIONS)
             .setInstanceId(memberId3)
             .setPreviousMemberEpoch(0)
-            .setTargetMemberEpoch(11)
             .setClientId("client")
             .setClientHost("localhost/127.0.0.1")
             .setRebalanceTimeoutMs(5000)
             .setSubscribedTopicNames(Arrays.asList("foo", "bar"))
             .setServerAssignorName("range")
-            .setPartitionsPendingAssignment(mkAssignment(
```

Comment:
do we have any way to track the partitions still pending? Or is it just a matter of the difference between the owned partition and the assignment?

Reply:
It is a matter of the difference between the owned partitions and the assignment. The field was not really used in the end so I thought that it is better to remove it to save space.